gtfs-realtime/spec/en/reference.md

```diff
@@ -297,6 +303,22 @@ For describing passenger occupancy levels on a linear scale, see `occupancy_perc
 | _**NO_DATA_AVAILABLE**_ | _The vehicle or carriage doesn't have any occupancy data available at that time._ |
 | _**NOT_BOARDABLE**_ | _The vehicle or carriage is not boardable and never accepts passengers. Useful for special vehicles or carriages (engine, maintenance carriage, etc…)._ |
 
+## _enum_ CrowdLevel
+
+The passenger crowd level for the vehicle or carriage. CrowdLevel enum values must be interpreted on an ordinal scale in increasing order from _**NOT_CROWDED**_ to _**CROWDED**_. In other words, producers must represent the lowest crowdedness level with _**NOT_CROWDED**_, the medium crowdedness level with _**SOME_CROWDING**_, and the highest crowdedness level with _**CROWDED**_. 
+
+To describe in-vehicle occupancy states, see [OccupancyStatus](#enum-occupancystatus). 
+
+Both OccupancyStatus and CrowdLevel may be used side-by-side, but must respectively indicate true conditions. For example, a vehicle may have _**FEW_SEATS_AVAILABLE**_ and _**SOME_CROWDING**_, have _**STANDING_ROOM_ONLY**_ and be _**CROWDED**_, or is _**NOT_ACCEPTING_PASSENGERS**_ and is _**CROWDED**_.
+
+**Caution:** this field is still **experimental**, and subject to change. It may be formally adopted in the future.
+
+| _**Value**_ | _**Comment**_ |
+|-------------|---------------|
+| _**NOT_CROWDED**_ | _The vehicle or carriage is not considered crowded by most measures, having few or no passengers onboard, or as otherwise defined by current operating regulations._ |
```

Comment:
Only the header text should be italicized, not the table body, although I see now it was done that way for OccupancyStatus. Both should be normal font in the "Value" and "Comment" section to match the other enum tables.

Reply:
As above, I copied the style precedent set by `OccupanyStatus`. If we change one we should change both, which I think is a good idea! It could be worth a separate project to clean up the GTFS Realtime reference style more globally.